azure-pipelines.yml

```diff
@@ -86,39 +73,15 @@ stages:
                terraform/**
                team-config/**
               TargetFolder: '$(Build.ArtifactStagingDirectory)'
-            condition: eq(variables['Build.SourceBranch'], 'refs/heads/master')
+            #condition: eq(variables['Build.SourceBranch'], 'refs/heads/master')
           - task: PublishBuildArtifacts@1
             displayName: 'Publish Artifact: drop'
-            condition: eq(variables['Build.SourceBranch'], 'refs/heads/master')
+            #condition: eq(variables['Build.SourceBranch'], 'refs/heads/master')
   - stage: Sandbox
-    condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
+    #condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
```

Comment:
Any reason removing condition for applying?

Reply:
testing, forgot to revert, will do before merge